src/Mod/Path/PathScripts/PathOpGui.py

```diff
@@ -1283,7 +1283,7 @@ def Create(res):
     obj = res.objFactory(res.name)
     if obj.Proxy:
         obj.ViewObject.Proxy = ViewProvider(obj.ViewObject, res)
-        obj.ViewObject.Visibility = False
+        obj.ViewObject.Visibility = True
```

Comment:
Is `True` not the default `Visibility` value?  If so, then this line may be removed all together, yes?